### PR TITLE
#Feat - Home Page

### PR DESCRIPTION
**Pull request by yogesh-**
Completed the homepage with the following sections:
- hero section
- categories
- trending videos

### CODE REVIEW

**Review by sashaboi — APPROVED** [review]
Looking forward to see future versions of this ! 
Code is very well written
Liked the folder structure
Looks good to merge.

**Review by sanjitsarkar — APPROVED** [review]
- Code looks clean and readable
- Simple and Clean UI
- Good to merge

File: src/components/banner/banner.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.banner {
+  display: flex;
+  flex-direction: row;
+  /* border: 2px solid red; */
+  margin: 0 4rem;
+  /* flex-flow: wrap; */
```
Comment (sharmayatendra) [inline src/components/banner/banner.css]: remove commented code if not needed

File: src/components/banner/banner.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.text-group :nth-child(2) {
+  font-size: 1.5rem;
+}
+
+.right-image img {
+  height: 500px;
+  width: 500px;
+}
```
Comment (sharmayatendra) [inline src/components/banner/banner.css]: use relative units

File: src/components/trending/trending.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+.video-container {
+  box-shadow: 0 1px 4px rgb(0 0 0 / 16%);
+  border-radius: 8px;
+  width: 16rem;
+  position: relative;
+}
+
+.video-container:hover {
+  transform: translateY(-5px);
+  border: 00.1rem solid black;
```
Comment (sharmayatendra) [inline src/components/trending/trending.css]: change it to 0.1 , 00.1 didn't make sense

File: src/components/trending/trending.css [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  padding-left: 10px;
+}
+
+.video-text {
+  display: flex;
+  flex-direction: row;
+}
+
+.channel-stats p {
+  color: #aaaaaa;
```
Comment (sharmayatendra) [inline src/components/trending/trending.css:61]: use color variables

**Review by sharmayatendra — APPROVED** [review]
good work
can look upon suggested changes

**Pull request merged**